cob_default_robot_config/cob3-2/base_configurations.yaml

```diff
@@ -1,2 +1,2 @@
-service_ns: /base_controller
-topic_name: /base_controller/command
+service_ns: base_controller
+topic_name: base_controller/command
```

Comment:
we need to be careful when we merge this vs. https://github.com/ipa320/cob_robots/pull/337